shapes/contacts/group.shex

```diff
@@ -0,0 +1,12 @@
+PREFIX vcard: <http://www.w3.org/2006/vcard/ns#>
+PREFIX xsd:   <http://www.w3.org/2001/XMLSchema#>
+
+vcard:Group {
+  vcard:fn xsd:string {1} ;
+  vcard:hasMember vcard:Individual * | vcard:Organization * # check this syntax 
+}
+
+# find out if there's a way to add this information as a backlink in the Group shape rather than having an AddressBook shape here
```

Comment:
I think this can be expressed using inverse triple constraints (see page 44 in https://www.slideshare.net/jelabra/shex-by-example-97678930), i.e. 

```
vcard:Group {
  ^vcard:includesGroup vcard:AddressBook ;
}
```